src/matrix/operator-python-module-py.cc

```diff
@@ -102,6 +102,10 @@ BOOST_PYTHON_MODULE(wrap)
   exposeUnaryOp < HomogeneousMatrixToVector>      ();
   exposeUnaryOp < SkewSymToVector>                ();
   exposeUnaryOp < PoseUThetaToMatrixHomo>         ();
+  exposeUnaryOp < HomogeneousMatrixToVector>      ();
```

Comment:
This one is already exposed 3 lines above.

Reply:
nice catch!

Reply:
except I can't use it:
```python
from dynamic_graph.sot.core import operator
operator.HomogeneousMatrixToVector()
```
→ `AttributeError: module 'dynamic_graph.sot.core.operator' has no attribute 'HomogeneousMatrixToVector'`

But that's working for others, like `HomoToMatrix()`. I don't see the difference.

Reply:
Ok, because that is renamed `MatrixHomoToPoseUTheta` in the bindings…

Reply:
That historic naming, we could just take the chance of renaming this properly here.